benchmarks/benchmark_throughput.py

```diff
@@ -270,9 +330,14 @@ def main(args: argparse.Namespace):
                                args.output_len)
     else:
         raise ValueError(f"Unknown backend: {args.backend}")
-    total_num_tokens = sum(prompt_len + output_len
-                           for _, prompt_len, output_len in requests)
-    total_output_tokens = sum(output_len for _, _, output_len in requests)
+    total_num_tokens = sum(request.prompt_len + request.expected_output_len
+                           for request in requests)
+    total_output_tokens = sum(
+        request.expected_output_len for request in requests)
+    if is_multi_model:
+        print("\033[91mWARNING\033[0m: Multi-modal request detected. "
+              "The following metrics is not accurate.")
```

Comment:
```suggestion
        print("\033[91mWARNING\033[0m: Multi-modal request detected. "
              "The following metrics is not accurate because image tokens are not counted. See vllm-project/vllm/issues/9778 for details")
```

Reply:
Done